criteria.md

```diff
@@ -0,0 +1,47 @@
+### Maturity Level 1 (Sandbox)
+ 
+| Category | Criteria Statement |
+|---|---|
+| Access Control | The project's version control system MUST require multi-factor authentication (MFA) for users when modifying the project repository settings or accessing sensitive data. |
+| Access Control | The project's version control system MUST restrict collaborator permissions to the lowest available privileges by default. |
+| Access Control | The project's version control system MUST prevent accidental direct commits against the primary branch. |
+| Access Control | The project's version control system MUST prevent accidental deletion of the default branch. |
+| Build & Release | The project's build and release pipelines MUST NOT execute arbitrary code that is input from outside of the build script. |
```

Comment:
| Build & Release | Project leads set schedule for document and whitepaper peer reviews. All documents and white papers MUST be published for pre-read and undergo peer review. after passing peer review, documents and whitepapers are published as final and versioned.
| Build & Release | Project leads set schedule for software peer reviews. All software releases MUST be published for pre-read and undergo peer review. after passing peer review, software is published as final and versioned.

Reply:
I like where you're going with this. Let's propose additions in a follow-up PR.